### ISSUE TITLE
App (and roku) crashes when selecting TV library from Home screen

### ISSUE THREAD

**Issue by cewert** [jellyfin-archive/jellyfin-roku-legacy#621]
**Describe the bug**
Viewing any TV library from the home screen crashes the app hard. so bad that my roku locks up. no home button. nothing. I have to hard reset the roku by pulling the power. ~~So this might be happening on other libraries but~~ this is a PITA to test. also happening on Movie libraries

**To Reproduce**

1. load latest version of dev
2. open a tv library from home -> my media
3. observe

**Expected behavior**
Works with no crash

**Logs**
```
Invalid value for left-side of expression. (runtime error &he4) in pkg:/source/ShowScenes.brs(340)
Backtrace:
#1  Function createitemgrid(libraryitem As Dynamic) As Dynamic
   file/line: pkg:/source/ShowScenes.brs(341)
#0  Function main(args As Dynamic) As Void
   file/line: pkg:/source/Main.brs(111)
Local Variables:
libraryitem      roSGNode:HomeData refcnt=2
global           Interface:ifGlobal
m                roAssociativeArray refcnt=3 count:7
group            Invalid
Threads:
ID    Location                                Source Code
 0*   pkg:/source/ShowScenes.brs(340)         group.parentItem = libraryItem
 1    ...ents/Buttons/TextSizeTask.brs(7)     reg = CreateObject("roFontRegistry")
  *selected
```

**Screenshots**
![crash-when-selecting-tv-library](https://user-images.githubusercontent.com/16514652/171004647-3e7a1ea0-6f2f-4867-81b7-3b4429dc803a.PNG)

**Additional context**
latest commit on dev

**Comment by cewert** [comment]
It would help if I knew how to read a log file :upside_down_face:

```
05-30 15:35:46.271 [beacon.signal] |AppLaunchComplete ---------> Pending Render Pass
05-30 15:35:46.300 [beacon.signal] |AppLaunchComplete ---------> Duration(2905 ms)
=================================================================
Error creating XML component ItemGrid
-- Attempt to add duplicate field "imageDisplayMode" to RokuML component "ItemGrid"
---- Extends node type "RenderableNode" already has a field named "imageDisplayMode"
-- Error foundpkg:/components/ItemGrid/ItemGrid.xml(42)
=================================================================
BRIGHTSCRIPT: ERROR: roSGNode: Failed to create roSGNode with type ItemGrid: pkg:/source/ShowScenes.brs(339)
```

**Comment by jimdogx** [comment]
Ah yes, I mentioned this on the Matrix Roku community channel 😄

**Comment by jimdogx** [comment]
Can we just fix this with this (#618) PR (remove the duplicate)?

**Comment by cewert** [comment]
We finally have our own channel! ~~Can you link me or something? I need to check this out 😄~~ I found it
